packages/main/src/ColorPalette.hbs

```diff
@@ -1,13 +1,26 @@
-<div class="ui5-cp-item-container" 
-	role="region"
-	aria-label="{{colorContainerLabel}}"
-	@click={{_onclick}}
-	@keyup={{_onkeyup}}
-	@keydown={{_onkeydown}}>
-	{{#each this.displayedColors}}
-		<slot
-			name="{{this._individualSlot}}"
-		>
-		</slot>
-	{{/each}}
+<div class="ui5-cp-root">
+	<div class="ui5-cp-item-container" 
+		role="region"
+		aria-label="{{colorContainerLabel}}"
+		@click={{_onclick}}
+		@keyup={{_onkeyup}}
+		@keydown={{_onkeydown}}>
+		{{#each this.displayedColors}}
+			<slot
+				name="{{this._individualSlot}}"
+			>
+			</slot>
+		{{/each}}
+	</div>
+
+	{{#if moreColors}}
+		<div class="ui5-cp-more-colors-wrapper">
+			<div class="ui5-cp-separator"></div>
+			<ui5-button
```

Comment:
Please check with @elenastoyanovaa if we need to add title and aria-label to this button, usually we do.

Reply:
not needed, since the button has text

Reply:
Oh, they are needed for icon-only button, I see. Thanks